doc/conf.py

```diff
@@ -21,7 +21,7 @@
 # Read package version without importing it
 for line in open(os.path.join(os.path.dirname(os.path.abspath('.')), "tarantool", "__init__.py")):
     if line.startswith("__version__"):
-        exec line
+        exec(line)
```

Comment:
A few words how do you meet a problem and how it looks would be useful. I would ever say that it is mandatory for this kind of changes. A problem may look obvious, but it does not. My initial guess was that the documentation building works only on sphinx that is run under Python 2 and it appears to be wrong. It worth to spend a minute to hold a problem and its appearance.

Surprisingly `python setup.py build_sphinx` works even for Python 3, but your change eliminates the following warning:

> WARNING: Support for evaluating Python 2 syntax is deprecated and will be removed in Sphinx 4.0. Convert /home/alex/projects/tarantool-meta/tarantool-python/doc/conf.py to Python 3 syntax.

So ironic: the tool is more backward compatible than the language it is written on.

BTW, I would add the 'doc' or the 'build' prefix (any of them looks applicable).

Reply:
I couldn't reproduce a problem with warning:
```
(venv) [s.bronnikov@tarantool-core-dev-mcs1 tarantool-python]$ python --version
Python 3.6.8
(venv) [s.bronnikov@tarantool-core-dev-mcs1 tarantool-python]$ python setup.py build_sphinx
running build_sphinx
Running Sphinx v3.3.1
loading pickled environment... done
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 0 source files that are out of date
updating environment: 0 added, 0 changed, 0 removed
looking for now-outdated files... none found
no targets are out of date.
build succeeded.

The HTML pages are in build/sphinx/html.
```
Before introducing a change I read official Python documentatio. `exec` was a statement in Python 2.x [1] and it has been converted to a function in Python 3.x [2], change introduced in version 3.0 [3]. So I just followed to modern Python style.

BTW warning happens when Sphinx failed to exec python code. It tries to perform conversion 2to3 and print a warning, see [4].

1. https://docs.python.org/2/reference/simple_stmts.html#exec
2. https://docs.python.org/3/library/functions.html#exec
3. https://docs.python.org/3/whatsnew/3.0.html
4. https://github.com/sphinx-doc/sphinx/blob/3.x/sphinx/util/pycompat.py#L68-L88

Reply:
And I think it would be good idea to include building Sphinx documentation in CI. Motivation the same as for tests and other actions in CI - check that documentation build passed. What do you think?

Reply:
So the motivation is purely to keep in track with Python 3 versions. I generally discourage changes, which do not directed by a problem, because of fear of subtle behaviour changes. But this one looks quite straightforward, so okay for me.

Regarding docs in CI: I completely okay. In the past I propose / implement CI targets of this kind in other projects:

https://github.com/tarantool/tarantool-java/issues/244
https://github.com/tarantool/graphql.0/pull/273